orbit/pkg/table/extension_darwin.go

```diff
@@ -70,7 +72,7 @@ func PlatformTables(opts PluginOpts) []osquery.OsqueryPlugin {
 		table.NewPlugin(
 			"sofa_security_release_info", sofa.SofaSecurityReleaseInfoColumns(),
 			func(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
-				return sofa.SofaSecurityReleaseInfoGenerate(ctx, queryContext, opts.Socket)
+				return sofa.SofaSecurityReleaseInfoGenerate(ctx, queryContext, opts.Socket, sofa.WithUserAgent("Fleet/"+build.Version))
```

Comment:
@lucasmrod will `build.Version` pull in the orbit version?  Thought that might be useful.

Reply:
Yes, possible, but that could mess up the server metrics? (Instead of seeing one fleetd user agent they will see multiple). And it's not always set (e.g. during development).

Maybe we can start without the version first.

Reply:
Makes sense, i'll omit a version for now.